backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -125,12 +109,54 @@ public void submitAccountRequest(@Valid @RequestBody AccountRequest body) throws
        */
       // send summary email to SR support
       String subject = "New account request";
-      _es.send(sendGridProperties.getAccountRequestRecipient(), subject, body);
+      _es.send(sendGridProperties.getAccountRequestRecipient(), subject, request);
       // send next-steps email to requester
-      _es.sendWithProviderTemplate(body.getEmail(), EmailProviderTemplate.ACCOUNT_REQUEST);
+      _es.sendWithProviderTemplate(request.getEmail(), EmailProviderTemplate.ACCOUNT_REQUEST);
 
-      createAdminUser(org, reqVars);
-      _crm.submitAccountRequestData(body);
+      createAdminUser(
+          request.getFirstName(), request.getLastName(), request.getEmail(), org.getExternalId());
+      _crm.submitAccountRequestData(request);
+    } catch (ResourceException e) {
+      // The `ResourceException` is thrown when an account is requested with an existing org
+      // name. This happens quite frequently and is expected behavior of the current form
+      throw e;
+    } catch (IOException | RuntimeException e) {
+      throw new AccountRequestFailureException(e);
+    }
+  }
+
+  /**
+   * Read the account request and generate an email body, then send with the emailService and create
+   * org without creating a facility
+   */
+  @SuppressWarnings("checkstyle:illegalcatch")
+  @PostMapping("/without-facility-with-emails")
+  @Transactional(readOnly = false)
+  public void submitAccountRequestWithoutFacility(
+      @Valid @RequestBody OrganizationAccountRequest request) throws IOException {
+    try {
+
+      logOrganizationAccountRequest(request);
+
+      Organization org = checkAccountRequestAndCreateOrg(request);
+
+      /**
+       * Note: we are sending the emails *after* creating the organization so the validation logic
+       * in the organization creation process functions as a CAPTCHA. But we are sending the emails
+       * *before* creating the user so that if sendgrid etc fails, we aren't left with an Okta
+       * entity created for the user that can't be automatically rolled back. (When we add automatic
+       * Okta rollback mechanisms down the road, this won't be an issue.)
+       */
+      // send summary email to SR support
+      String subject = "New account request";
+      _es.send(sendGridProperties.getAccountRequestRecipient(), subject, request);
+      // send next-steps email to requester
```

Comment:
nit: since all this send-email work is shared between this endpoint and the original endpoint, would you consider moving it out to a separate function?